scss/_patterns_pagination.scss

```diff
@@ -43,6 +43,15 @@
     &--truncation {
       padding: $input-vertical-padding 0;
     }
+
+    a {
+      &.is-active,
+      &[aria-current='page'] {
+        background-color: $colors--theme--background-active;
```

Comment:
@britneywwc Why are these separate from active styles in lines 8-12? I guess this should replace the hardcoded colours there?

Reply:
I must've missed it, thanks! I moved the themed styles to replace the active styles.